src/implementation/Client/GRPCClient/invoker.ts

```diff
@@ -47,7 +47,7 @@ export default class GRPCClientInvoker implements IClientInvoker {
       };
     }
 
-    if (method !== HttpMethod.GET && data !== {}) {
```

Comment:
Any specific reason for including an empty body in `fetchOptions`?

Reply:
It breaks typescript as this doesn't do anything in Javascript